components/Input/InputGroup.tsx

```diff
@@ -11,13 +11,15 @@ interface IInputGroup extends Omit<React.InputHTMLAttributes<HTMLInputElement>,
 interface IInputWrapper {
 	iconSize?: number;
 	iconPosition: 'right' | 'left';
+	disabled?: boolean;
 }
 
 export const InputGroup: FC<IInputGroup> = ({ icon, iconPosition = 'right', ...rest }) => {
 	const iconRef = useRef<HTMLInputElement>();
 
 	return (
 		<InputWrapper
+			disabled={rest.disabled}

```

Comment:
lets take this out of rest if we are passing it in here

Reply:
Done